src/router/constants/authRoutes.ts

```diff
@@ -51,6 +51,9 @@ export const authRoutes = {
   cooperationQuiz: {
     route: 'my-cooperations/:id/quiz/:quizId'
   },
+  cooperationQuizPreview: {
+    route: 'my-cooperations/:id/preview/:quizId'
```

Comment:
What is the preview? It's just a `quiz` but in another mode. So you can add a query parameter for that (`mode=preview`) and don't add an additional route for that. If it's not possible - add `preview` to the end of the route.
```suggestion
    route: 'my-cooperations/:id/quiz/:quizId/preview'
```

Reply:
![image](https://github.com/user-attachments/assets/b8a8b8ae-c263-4250-8ef9-4f793fc5b16e)

so preview is this page

and the other route where we have quiz there is

![image](https://github.com/user-attachments/assets/cc24f476-68c7-433f-9316-acf9940c3959)

so it's 2 whole separate pages how it was supposed to be from the start. But one of our team mates messed up and didn't do so

http://localhost:3000/my-cooperations/6799352c7272c561ede2cf0e/`quiz`/67ade0347c0afe4bcb604cf6
http://localhost:3000/my-cooperations/6799352c7272c561ede2cf0e/`preview`/67ade0347c0afe4bcb604cf6

so I thought it would make the most sense if we have quiz in the same place to have preview there

Reply:
First of all, your URLs are really bad. It's not obvious at all, that `my-cooperations/:id/preview` - is a preview of Quiz. You should change it, because a lot of information that user can receive from page - holds in URL.

I would suggest you to make it like this:

- `my-cooperations/:cooperationId/quizzes` - all quizzes for this cooperation
- `my-cooperations/:cooperationId/quizzes/:quizId` - particular quiz in cooperation (where user can start new attempt and answer on every question of quiz)
- `my-cooperations/:cooperationId/quizzes/:quizId/attempts` - all attempts of particular quiz (your second pic)
- `my-cooperations/:cooperationId/quizzes/:quizId/attempts/:attemptId`- previewing particular attempt of particular quiz